### PR TITLE
LukeHjo's Awesome Branch

### PR DESCRIPTION
**Pull request by luke-beep**
Added "Increase icon cache size" under MemOptimization.

Added "disablewriteautotiering" using fsutil under MemOptimization.
Increased "quotanotify" to 5400 seconds using fsutil under MemOptimization.

### CODE REVIEW

File: HoneCtrl.bat [hunk 1]
```diff
@@ -122,8 +122,9 @@ if "%firstlaunch%" == "0" (goto MainMenu)
 
 REM Restore Point
 reg add "HKLM\Software\Microsoft\Windows NT\CurrentVersion\SystemRestore" /v "SystemRestorePointCreationFrequency" /t REG_DWORD /d 0 /f >nul 2>&1
-powershell -ExecutionPolicy Unrestricted -NoProfile Enable-ComputerRestore -Drive 'C:\', 'D:\', 'E:\', 'F:\', 'G:\' >nul 2>&1
-powershell -ExecutionPolicy Unrestricted -NoProfile Checkpoint-Computer -Description 'Hone Restore Point' >nul 2>&1
+REM powershell -ExecutionPolicy Unrestricted -NoProfile Enable-ComputerRestore -Drive 'C:\', 'D:\', 'E:\', 'F:\', 'G:\' >nul 2>&1
```
Comment (JonathahWasTaken) [inline HoneCtrl.bat]: Uncomment this and we good

**Review by JonathahWasTaken — CHANGES_REQUESTED** [review]

File: HoneCtrl.bat (hunk 1 above)
Comment (luke-beep) [reply HoneCtrl.bat]: Alright.

**Review by JonathahWasTaken — APPROVED** [review]

**Comment by couleurm** [comment]
why are you not using the SYSTEMDRIVE environment variable

**Comment by luke-beep** [comment]
For what exactly?

**Comment by luke-beep** [comment]
Setting it to the \\root\default namespace is the best practice for most WMI(C) operations.

**Pull request merged**